aspnetcore/blazor/call-javascript-from-dotnet.md

```diff
@@ -514,6 +514,8 @@ public async ValueTask<string> Prompt(string message)
 }
 ```
 
+`IJSInProcessObjectReference` represents a reference to a JavaScript object whose functions can be invoked synchronously.
```

Comment:
Can we also add a snippet about `IJSUnmarshalledObjectReference`

`IJSUnmarshalledObjectReference` represents a reference to an JavaScript object whose functions can be invoked without the overhead of serializing .NET data. This can be used in Blazor WebAssembly when performance is crucial:

```js
window.unmarshalledInstance = {
 helloWorld: function (personNamePointer) {
      const personName = Blazor.platform.readStringField(value, 0);
      return `Hello ${personName}`;
 }
};
```

```C#
var unmarshalledRuntime = (IJSUnmarshalledRuntime)jsRuntime;
var jsUnmarshalledReference = unmarshalledRuntime.InvokeUnmarshalled<IJSUnmarshalledObjectReference>("unmarshalledInstance");

string helloWorldString = jsUnmarshalledReference.InvokeUnmarshalled<string, string>("helloWorld");
```

I haven't tested this out, but I think this should work.

Reply:
Sure ... I'll add that on the next commit.

Did you have anything in addition for `IJSInProcessObjectReference` :point_up:?